### ISSUE TITLE
crafter selection: auto craft on the lowest levelled crafter

### ISSUE THREAD

**Issue by Jaksuhn** [awgil/ffxiv_satisfy#1]
it's good xp if you're still levelling them. Instead of defaulting to crp (or first doh gearset, didn't look at what you do), find the lowest levelled one that would still be able to craft it

**Comment by awgil** [comment]
Good idea, i'll probably add a selector + some auto-selection options. Some considerations:
* getting xp vs getting currency - conceivably user might prefer either
* when choosing among L100 crafters, i believe crystals are different - might prefer using ones you have biggest amount of

Currently it always uses CRP.

**Comment by Jaksuhn** [comment]
added this as a limited option
ideally there could be an artisan ipc check to see if a recipe can be crafted with a given job or something to prevent it getting stuck with some weird level set someone might have and will complain about
